hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieIndex.java

```diff
@@ -616,7 +616,7 @@ public void testDelete() throws Exception {
     assertEquals(records.size() - numDeletes, javaRDD.map(record -> record.getKey().getRecordKey()).distinct().count());
   }
 
-  private HoodieWriteConfig.Builder getConfigBuilder() {
+  public HoodieWriteConfig.Builder getConfigBuilder() {
```

Comment:
why these changes? any test methods used across test classes, should go into a utils method or some base test method.  curious to know why do we need to make this public ?

Reply:
This is because it's overriding the method in the base class now